openslides/core/static/css/core/_countdown.scss

```diff
@@ -24,9 +24,3 @@
     font-weight: normal;
     font-size: 5vw;
 }
-#countdown.warning {
```

Comment:
Please move the countdown related rules here.

Reply:
done